src/main/java/homework3/ex1/page/Ex1TopNavigationBar.java

```diff
@@ -0,0 +1,53 @@
+package homework3.ex1.page;
+
+import homework3.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class Ex1TopNavigationBar extends BasePage {
+
+    @FindBy(css = ".m-l8 > li")
+    private List<WebElement> topNavBar;
+    @FindBy(css = ".m-l8 > li:nth-child(1)")
+    private WebElement homeTopNavBtn;
+    @FindBy(css = ".m-l8 > li:nth-child(2)")
+    private WebElement contactTopNavBtn;
+    @FindBy(css = ".m-l8 > li:nth-child(3)")
+    private WebElement serviceTopNavBtn;
+    @FindBy(css = ".m-l8 > li:nth-child(4)")
+    private WebElement metalAndColorsTopNavBtn;
+
+    public Ex1TopNavigationBar(WebDriver driver) {
+        super(driver);
+    }
+
+    public Ex1HomePage topNavBtnHaveProperNumberAndAreDisplayed(int expectedAmount) {
+        elementsHaveProperNumber(topNavBar, expectedAmount);
+        allElementsAreDisplayed(topNavBar);
+        return new Ex1HomePage(driver);
+    }
+
+    public Ex1HomePage topHomeBtnHasProperText(String expectedText) {
+        equalsText(homeTopNavBtn, expectedText);
+        return new Ex1HomePage(driver);
```

Comment:
**pls make elements PO methods return WebElement and use it in appropriate page**

it's very bad to return every time new page